docs/operators/s3outputmodule.md

```diff
@@ -0,0 +1,207 @@
+    -   **S3BlockUploadOperator** emits the pair **(path, ETag)** to **s3FileMerger**.
+
+-   ***S3FileMerger***
+    -   Complete multipart upload request using **AmazonS3Client.completeMultipartUpload(...)**. This call must include the **upload ID** and a list of both part numbers and corresponding **ETag** values. **S3FileMerger** sends the complete multi-part upload request to S3 once it has all the **part ETag's** of a file. 
+**Amazon S3** creates an object by concatenating the parts in ascending order based on part number. After a successful upload request, the parts no longer exist and S3 response includes an **ETag** which uniquely identifies the combined object data. 
+
+### Configuration Parameters
+-----------------------------
+
+-   ***accessKey*** -   String
+    -   Mandatory Parameter    
+    -   Specifies the AWS access key to access Amazon S3 and has permissions to access the specified bucket.
+    -   Example value = AKIAJVAGFANC2LSZCJ4Q
+
+-   ***secretAccessKey***   -   String
+    -   Mandatory Parameter
+    -   Specifies the AWS secret access key to access Amazon S3 and has permissions to access the specified bucket.
+    -   Example value = wpVr3U82RmCKJoY007YfkaawT7CenhTcK1B8clue
+    
+-   ***endPoint***  -   String
+    -   Endpoint is the URL for the entry point for a web service. Specify the valid endpoint to access S3 bucket.
+    -   This is an optional parameter. If the bucket is accessed only from specific end point then the user has to specify this parameter.
+    -   Please refer to [endPoint](http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region) table about the endpoints supported by S3. 
+    -   Example value = s3.amazonaws.com 
+
+-   ***bucketName***    -   String
+    -   Mandatory Parameter
+    -   S3 buckets are used to store objects which consists of data and metadata that describes the data. Specify the name of the bucket.
+    -   Example value = apex.app.test.s3
+
+-   ***outputDirectoryPath***   -   String
+    -   Mandatory Parameter
+    -   Specifies the path of the output directory. 
+    -   Example value = dt/app/output
+
+-   ***mergerCount***   -   int
+    -   Specify the number of instances of S3FileMerger operator.
+    -   Default value = 1
+    
+-   ***timeOutWIndowCount***    -   int
+    -   This property maps to the (OperatorContext.TIMEOUT_WINDOW_COUNT)[https://github.com/apache/apex-core/blob/master/api/src/main/java/com/datatorrent/api/Context.java) attribute and is a count of streaming windows. If specified, it will be set on all the operators of this module. Since these operators interact with S3, there may be additional latencies that cause the platform to kill them because they are considered stalled. Increasing this value prevents this and allows the application to proceed despite the latencies.
```

Comment:
This link is not properly formatted: should be:
[OperatorContext.TIMEOUT_WINDOW_COUNT](https://github.com/apache/apex-core/blob/master/api/src/main/java/com/datatorrent/api/Context.java)